spring-pulsar-docs/src/main/antora/modules/ROOT/pages/reference/reactive-pulsar.adoc

```diff
@@ -163,8 +163,7 @@ In this most basic form, you must still provide the topic name by setting the fo
 
 [source,yaml,indent=0,subs="verbatim"]
 ----
-spring.pulsar.reactive.consumer:
-  topic-names: hello-pulsar-topic
+spring.pulsar.reactive.consumer.topic-names: hello-pulsar-topic
```

Comment:
This should be `spring.pulsar.consumer.topics` - from what I see both imperative and reactive share the same configuration property for this.

Reply:
Ditto from https://github.com/spring-projects/spring-pulsar/pull/482#discussion_r1389522710